csm/core/controllers/login.py

```diff
@@ -70,4 +70,5 @@ async def post(self):
         await self.request.app.login_service.logout(session_id)
         # TODO: Stop any websocket connection corresponding to this session
         Log.info('Session ended')
+        Log.info(f"user_id: {self.request.session.credentials.user_id} successfully logged out")
```

Comment:
can we merge these two logs into one ?  'Session ended' is too generic and not giving information about the user for which session ended etc. and in the logs it is not not looking good. ( refer dev testing screenshot )

Reply:
Addressed